leftwm-core/src/utils/command_pipe.rs

```diff
@@ -90,38 +90,45 @@ async fn read_from_pipe(pipe_file: &Path, tx: &mpsc::UnboundedSender<Command>) -
 fn parse_command(s: &str) -> Result<Command, Box<dyn std::error::Error>> {
     let (head, rest) = s.split_once(' ').unwrap_or((s, ""));
     match head {
-        "SoftReload" => Ok(Command::SoftReload),
-        "ToggleFullScreen" => Ok(Command::ToggleFullScreen),
-        "ToggleSticky" => Ok(Command::ToggleSticky),
-        "SwapScreens" => Ok(Command::SwapScreens),
-        "MoveWindowToNextTag" => build_move_window_to_next_tag(rest),
-        "MoveWindowToPreviousTag" => build_move_window_to_previous_tag(rest),
```

Comment:
The only thing is that MoveWindowToNext/PreviousTag have been accidentally removed.

Reply:
Thanks